projects/demo/src/app/pages/slash/index.html

```diff
@@ -0,0 +1,19 @@
+<tui-doc-page
+    header="Editor"
+    type="components"
+>
+    <tui-doc-example
+        id="slash"
+        heading="Slash commands"
+        [component]="component1"
+        [content]="example1"
+        [description]="description"
+    >
+        <ng-template #description>
+            <p>
+                The Slash example adds a toolbar via / that pops up at the slash position, well known from other editor
+                applications. It's great to quickly insert different content types or apply inline formatting.
+            </p>
```

Comment:
Do we really need `<p />` ?

```suggestion
                The Slash example adds a toolbar via / that pops up at the slash position, well known from other editor
                applications. It's great to quickly insert different content types or apply inline formatting.
```